wit/ephemeral/redis-types.wit

```diff
@@ -6,3 +6,11 @@ enum error {
 
 // The message payload.
 type payload = list<u8>
+
+// An argument and return type for the general-purpose `execute` function.
+variant value {
+    nil,
+    %string(string),
```

Comment:
We're of course under no obligation to be consistent, and it may be more appropriate to use Redis type names for a product-specific instance like this, but the database WIT uses the names `str`, `int64` and `binary` for the variant cases.  Obviously your call as to whether alignment is appropriate or not.